app/templates/browser-support.hbs

```diff
@@ -3,32 +3,10 @@
   <section aria-labelledby="section-browser-support-policy">
     <h1 id="section-browser-support-policy">Ember.js Browser Support Policy</h1>
 
-    <h2>Ember 3.0.0</h2>
-
-    <p>
-      Ember currently targets Internet Explorer 11 as a baseline for support. This means that generally all modern and relatively recent browsers will work with Ember, since browsers are backwards compatible by design. Ember runs tests against the latest desktop versions of the following browsers:
-    </p>
-
-
-
-    <div class="layout my-3">
-      <div class="card lg:col-2 lg:start-3">
-        <div class="card__content">
-          <ul>
-            <li>Google Chrome</li>
-            <li>Mozilla Firefox</li>
-            <li>Microsoft Edge</li>
-            <li>Internet Explorer 11</li>
-            <li>Safari</li>
-          </ul>
-        </div>
-      </div>
-    </div>
-
-    <h2>Ember 4.0.0</h2>
+    <h2>Ember 4.0.0 and Ember 5.0.0</h2>
```

Comment:
moved v4 and v5 to the top as that's what I think most ppl would like to see first